src/components/buttons/rainbow-button/RainbowButtonBackground.js

```diff
@@ -43,18 +43,20 @@ const RainbowButtonGradient = styled(RadialGradient).attrs(
     stops:
       type === RainbowButtonTypes.addCash ? [0, 0.544872, 1] : [0, 0.774321, 1],
   })
-)`
-  position: absolute;
-  transform: scaleY(0.7884615385);
-`;
+)({
+  position: 'absolute',
+  transform: [{ scaleY: 0.7884615385 }],
```

Comment:
```suggestion
  transform: [{ scaleY: 0.7884615384 }],
```

I believe this looks significantly better, cc @christianbaroni